api/python/quilt3/data_transfer.py

```diff
@@ -298,10 +298,15 @@ def upload_part(i, start, end):
                     Bucket=dest_bucket,
                     Key=dest_key,
                     UploadId=upload_id,
-                    PartNumber=part_id
+                    PartNumber=part_id,
+                    ChecksumAlgorithm='SHA256',
                 )
             with lock:
-                parts[i] = {"PartNumber": part_id, "ETag": part["ETag"]}
+                parts[i] = dict(
+                    PartNumber=part_id,
+                    ETag=part["ETag"],
+                    ChecksumSHA256=part['ChecksumSHA256'],
```

Comment:
Why single quotes here and double quotes on the line above?

Reply:
Same comment for a few other places.

Reply:
Oh, didn't notice. Don't think there's an official style, but I like to follow [this](https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python):

> double quotes around strings that are used for interpolation or that are natural language messages, and single quotes for small symbol-like strings